pyanaconda/kickstart.py

```diff
@@ -80,12 +80,32 @@
 from pykickstart.version import returnClassForVersion
 
 import logging
-log = logging.getLogger("anaconda")
+anaconda_log = logging.getLogger("anaconda")
+script_log = logging.getLogger("anaconda.kickstart.script")
+parsing_log = logging.getLogger("anaconda.kickstart.parsing")
 stderrLog = logging.getLogger("anaconda.stderr")
 storage_log = logging.getLogger("blivet")
 stdoutLog = logging.getLogger("anaconda.stdout")
 from pyanaconda import anaconda_log
 
+# command specififc loggers
+authconfig_log = logging.getLogger("anaconda.kickstart.authconfig")
```

Comment:
I like the statically defined loggers on top of a file per module, but I don't like the loggers for kickstart commands, because I think we should treat them the same way as the class loggers (in the future). Then the `self.log` might be a way to go. Good point about the serialization ;-)